.circleci/config.yml

```diff
@@ -459,15 +459,13 @@ jobs:
           command: |
             #
             # This block should not cause a test failure and block PRs.
-            # The shell should never error and exit 0 to indicate success.
             #
             set +o errexit
-            npm run build-css                                      \
-              && npm run preanalyze-bundles                        \
-              && node bin/icfy-analyze.js                          \
-              && mv client/stats.json client/chart.json client/style.json "$CIRCLE_ARTIFACTS/icfy" \
+            npm run build-css                                                                       \
```

Comment:
We no longer need to run the `build-css` job, as it doesn't produce any stylesheed we'd want to analyze. We used to analyze `public/style.css`, but it was all migrated away to webpack.